prc/ecdata/constancies.tex

```diff
@@ -39,6 +39,7 @@
         \item \ecdataPhase{}
         \item \indexMax{}
         \item \notOnGTwoAccMax{}
+        \item \If $\isEcpairingResult_{i} = 1$ \Then \trivialPairing{}
```

Comment:
This does not work here. This should be moved to whatever section imposes general constraints on `TRIVIAL_PAIRING`